accounts/views.py

```diff
@@ -72,5 +78,14 @@ def register(request):
 
 def user_profile(request):
     """The user's profile page"""
-    user = User.objects.get(email=request.user.email)
-    return render(request, 'profile.html', {'profile': user})
\ No newline at end of file
+    
+    user = request.user
+    quotes = Quote.objects.filter(order__customer=user.id)
+    number_of_quotes = quotes.count()
+    quote_numbers = []
+    for i in range(1, number_of_quotes+1):
+        quote_numbers.append(str(i))
+    quotes_zip = zip(quote_numbers, quotes)
```

Comment:
the python built-in function - enumerate - gives you this for free.
https://docs.python.org/3/library/functions.html#enumerate
just pass in quotes only and use enumerate in the template when looping to get the index.
pass start = 1 so index starts from 1 instead of 0

Reply:
Resolved. {{ forloop.counter }} used in template.